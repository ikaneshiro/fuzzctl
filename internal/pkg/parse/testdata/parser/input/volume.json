{
  "version": "0.1",
  "workflow": {
    "name": "Volumes",
    "jobs": [
      {
        "name": "date",
        "image": "library://alpine:latest",
        "command": [
          "date",
          ">",
          "/data/time"
        ],
        "volumes": [
          {
            "name": "v1",
            "location": "/data"
          }
        ]
      },
      {
        "name": "hello",
        "image": "library://alpine:latest",
        "command": [
          "echo",
          "-n",
          "Hello, world, the time is $(cat /share/time)"
        ],
        "requires": [
          "date"
        ],
        "volumes": [
          {
            "name": "v1",
            "location": "/share"
          }
        ]
      }
    ],
    "volumes": [
      {
        "name": "v1",
        "type": "ephemeral"
      }
    ]
  }
}