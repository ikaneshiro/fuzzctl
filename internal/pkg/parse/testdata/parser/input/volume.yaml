version: 0.1
workflow:
  name: Volumes
  jobs:
    date:
      image: library://alpine:latest
      command: ["date", ">", "/data/time"]
      volumes:
        v1:
          location: "/data"
    hello:
      image: library://alpine:latest
      command: ["echo", "-n", "Hello, world, the time is $(cat /share/time)"]
      requires: ["date"]
      volumes:
        v1:
          location: "/share"
  volumes:
    v1:
      type: "ephemeral"