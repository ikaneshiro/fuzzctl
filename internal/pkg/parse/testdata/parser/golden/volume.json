{
	"name": "Volumes",
	"jobs": [
		{
			"name": "date",
			"image": "library://alpine:latest",
			"command": [
				"date",
				"\u003e",
				"/data/time"
			],
			"requires": null,
			"volumes": [
				{
					"name": "v1",
					"location": "/data"
				}
			]
		},
		{
			"name": "hello",
			"image": "library://alpine:latest",
			"command": [
				"echo",
				"-n",
				"Hello, world, the time is $(cat /share/time)"
			],
			"requires": [
				"date"
			],
			"volumes": [
				{
					"name": "v1",
					"location": "/share"
				}
			]
		}
	],
	"volumes": [
		{
			"name": "v1",
			"type": "ephemeral"
		}
	]
}